pkgs/data/themes/matcha/default.nix

```diff
@@ -25,8 +25,8 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    description = "A stylish Design theme for GTK based desktop environments";
-    homepage = "https://vinceliuice.github.io/theme-matcha";
+    description = "A stylish flat Design theme for GTK based desktop environments";
+    homepage = "https://github.com/vinceliuice/Matcha-gtk-theme";
```

Comment:
The previous URL still contains a nice web page with lots of screenshots. I'd keep that one.

Reply:
Ok. It will be kept.